src/CSV_Parser.php

```diff
@@ -42,6 +46,7 @@ class CSV_Parser {
 	 */
 	public function __construct() {
 		$this->setData(null);
+		$this->setEncoding(null);
```

Comment:
I can't think of a good enough reason not to unless it's making too much of an assumption. PHP 5.6 and beyond default encoding to the setting of php.ini's _default_charset_ (earlier uses _mbstring.internal_encoding_). The default setting for _default_charset_ is UTF-8, so it would make sense to go with that.